app/Http/Controllers/ProjectController.php

```diff
@@ -27,6 +27,9 @@ public function mass_update(Request $request)
 
         // Insert Eloquent statement below
 
+        Project::where('name', $request->old_name)
```

Comment:
You can use `whereName($request->old_name)` as well

Reply:
Yep, but I like this way more.